userprofile/api/serializers.py

```diff
@@ -26,12 +25,27 @@ class UserSerializer(UserBriefSerialiser):
     Add the details of a user.
     """
 
+    courses = serializers.SerializerMethodField('list_courses')
```

Comment:
Something like `courses = CourseBriefSerializer(source='enrolled', many=True)` should be enough